src/Commands/AzureISV/Datadog/MonitoredResourcesListCommand.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.CommandLine;
+using System.CommandLine.Parsing;
+using AzureMcp.Arguments.AzureISV.Datadog;
+using AzureMcp.Commands.AzureISV.Datadog;
+using AzureMcp.Models.Argument;
+using AzureMcp.Models.Command;
+using AzureMcp.Services.Interfaces;
+using Microsoft.Extensions.Logging;
+
+namespace AzureMcp.Commands.Datadog.MonitoredResources;
+
+public sealed class MonitoredResourcesListCommand(ILogger<MonitoredResourcesListCommand> logger) : SubscriptionCommand<MonitoredResourcesListArguments>()
+{
+    private readonly ILogger<MonitoredResourcesListCommand> _logger = logger;
+
+    protected override string GetCommandName() => "list";
```

Comment:
Please rebase with main.  We update the BaseCommand interface. Make sure you add the title property as well and update the McpServerTool attribute with the title.